lib/statsd/instrument/datagram_builder.rb

```diff
@@ -25,31 +59,35 @@ def initialize(prefix: nil, default_tags: nil)
       end
 
       def c(name, value, sample_rate, tags)
-        generate_generic_datagram(name, value, "c", sample_rate, tags)
+        generate_generic_datagram(name, [value], "c", sample_rate, tags)
       end
 
       def g(name, value, sample_rate, tags)
-        generate_generic_datagram(name, value, "g", sample_rate, tags)
+        generate_generic_datagram(name, [value], "g", sample_rate, tags)
       end
 
       def ms(name, value, sample_rate, tags)
-        generate_generic_datagram(name, value, "ms", sample_rate, tags)
+        generate_generic_datagram(name, [value], "ms", sample_rate, tags)
       end
 
       def s(name, value, sample_rate, tags)
-        generate_generic_datagram(name, value, "s", sample_rate, tags)
+        generate_generic_datagram(name, [value], "s", sample_rate, tags)
       end
 
       def h(name, value, sample_rate, tags)
-        generate_generic_datagram(name, value, "h", sample_rate, tags)
+        generate_generic_datagram(name, [value], "h", sample_rate, tags)
```

Comment:
Passing an array here is an extra allocation for every call, wouldn't it be simpler to do the `.join(":")` in `timing_value_packed ` and leave `generate_generic_datagram` unchanged?